README.md

```diff
@@ -393,7 +393,7 @@ public class FluxExample {
 
 ## Build Requirements
 
-* Java 1.8+ (tested with jdk8)
+* Java 17+ (tested with JDK 17)
```

Comment:
I know this is a draft, but I wanted to put this down so I don't forget. Can we add a note for users who are stuck on JDK 8 and JDK 11, to use the previous version?

Reply:
Of course, I will fix the tests and then update the CHANGELOG and README documentation.

Reply:
done